### PR TITLE
add landing message #1

### PR DESCRIPTION
**Pull request by cnguyen64atstlcc**
just add <h1>

### CODE REVIEW

**Review by chgngyn — COMMENTED** [review]
please add color and remove text

**Comment by cnguyen64atstlcc** [comment]
ok, I will add color and remove the text!

**Review by chgngyn — CHANGES_REQUESTED** [review]
please remove text h1

File: index.html
Comment (chgngyn) [inline index.html:1]: ad h2 instead of h1

Comment (cnguyen64atstlcc) [inline index.html:1]: ok sir

**Review by chgngyn — APPROVED** [review]

**Pull request merged**